webpack/scenes/ActivationKeys/Details/ActivationKeyDetails.js

```diff
@@ -1,10 +1,114 @@
+import DeleteMenu from './components/DeleteMenu';
+import { getActivationKey } from './ActivationKeyActions';
+import DeleteModal from './components/DeleteModal';
 
-const ActivationKeyDetails = ({ match }) => <div>ActivationKeyDetails { match?.params?.id } </div>;
+const ActivationKeyDetails = ({ match }) => {
+  const dispatch = useDispatch();
+  const akId = match?.params?.id;
+  const akDetailsResponse = useSelector(state => selectAPIResponse(state, `ACTIVATION_KEY_${akId}`));
+  const akDetails = propsToCamelCase(akDetailsResponse);
+  useEffect(() => {
+    if (akId) { // TODO add back akNotLoaded condition
+      dispatch(getActivationKey(akId));
+    }
+  }, [akId, dispatch]);
+
+  const [isModalOpen, setModalOpen] = useState(false);
+  const handleModalToggle = () => {
+    setModalOpen(!isModalOpen);
+  };
+
+  return (
+    <div >
+      <Panel className="ak-details-header">
+        <div className="breadcrumb-bar-pf4">
+          <Breadcrumb ouiaId="ak-breadcrumbs" className="breadcrumb-display">
+            <BreadcrumbItem className="breadcrumb-list" to="/activation_keys">
+              {__('Activation keys')}
+            </BreadcrumbItem>
+            <BreadcrumbItem to="#" isActive>
+              {akDetails.name}
+            </BreadcrumbItem>
+          </Breadcrumb>
+        </div>
+        <Grid>
+          <GridItem span={8} className="ak-name-wrapper">
+            <Flex justifyContent={{ default: 'jusifyContentSpaceBetween' }} alignItems={{ default: 'alignItemsCenter' }}>
+              <FlexItem>
+                <Title ouiaId="ak-title" headingLevel="h5" size="2xl" className="ak-name-truncate">
+                  {akDetails.name}
+                </Title>
+              </FlexItem>
+              <FlexItem>
+                <Split hasGutter style={{ display: 'inline-flex' }}>
+                  <SplitItem>
+                    <Label>
+                      {akDetails.usageCount ? akDetails.usageCount : 0}/{akDetails.unlimitedHosts ? __('Unlimited') : akDetails.maxHosts}
+                    </Label>
+                  </SplitItem>
+                </Split>
+              </FlexItem>
+            </Flex>
+          </GridItem>
+          <GridItem offset={8} span={4}>
+            <Flex>
+              <FlexItem align={{ default: 'align-right' }}>
+                <Split>
+                  <SplitItem>
+                    <EditModal akDetails={akDetails} akId={akId} />
+                  </SplitItem>
+                  <DeleteMenu handleModalToggle={handleModalToggle} akId={akId} />
+                </Split>
+              </FlexItem>
+            </Flex>
+          </GridItem>
+        </Grid>
+        <div className="ak-details-description">
+          <TextContent>
+            <Text ouiaId="ak-description" component={TextVariants.p}>
+              {akDetails.description ? akDetails.description : <span style={{ color: '#c1c1c1' }}>{__('No description provided')}</span>}
```

Comment:
Could you try <InactiveText> component for this. It lives under ContentViews/components but we can reuse that. Don't have to do it if it doesn't render right.